tests/PubmedIdTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Altmetric\Identifiers;
+
+class PubmedIdTest extends \PHPUnit_Framework_TestCase
+{
+    public function testExtractsPubMedIDs()
```

Comment:
We should pick a consistent capitalisation for PubMed: as the class is `Pubmed`, shall we stick with that?